sickbeard/__init__.py

```diff
@@ -1622,7 +1622,7 @@ def save_config():
     new_config['General']['config_version'] = CONFIG_VERSION
     new_config['General']['encryption_version'] = int(ENCRYPTION_VERSION)
     new_config['General']['encryption_secret'] = ENCRYPTION_SECRET
-    new_config['General']['log_dir'] = ACTUAL_LOG_DIR if ACTUAL_LOG_DIR else 'Logs'
+    new_config['General']['log_dir'] = ACTUAL_LOG_DIR if ACTUAL_LOG_DIR else 'logs'
     new_config['General']['log_nr'] = int(LOG_NR)
```

Comment:
For the same reason the python dirs are called lib, sickbeard and sickrage, instead of Lib, SickBeard and SickRage: It's just good practice to do all lowercase in dirnames when being cross platform/ cross-fs.

Besides that, it's in a number of LFS best practices that I know of, and I'm sure some others that I don't know of.

-----Original Message-----
From: "Fernando" <notifications@github.com>
Sent: ‎9/‎26/‎2015 19:18
To: "SiCKRAGETV/SickRage" <SickRage@noreply.github.com>
Cc: "Nils" <nils@familievogels.nl>
Subject: Re: [SickRage] Change default DATA_DIR to PROG_DIR/data (#2561)

In sickbeard/__init__.py:
> @@ -1622,7 +1622,7 @@ def save_config():
>      new_config['General']['config_version'] = CONFIG_VERSION
>      new_config['General']['encryption_version'] = int(ENCRYPTION_VERSION)
>      new_config['General']['encryption_secret'] = ENCRYPTION_SECRET
> -    new_config['General']['log_dir'] = ACTUAL_LOG_DIR if ACTUAL_LOG_DIR else 'Logs'
> +    new_config['General']['log_dir'] = ACTUAL_LOG_DIR if ACTUAL_LOG_DIR else 'logs'
>      new_config['General']['log_nr'] = int(LOG_NR)
why logs and not Logs?
—
Reply to this email directly or view it on GitHub.